### DISCUSSION TITLE
Lien Drive en streaming ?

### DISCUSSION THREAD

**Discussion by Jeanaimarre** [Kodi-vStream/venom-xbmc-addons#3982] [Idées]
Hello
J'aimerais savoir si il serait possible d'ajouter une fonctionnalité pour ajouter son compte google à vstream et ainsi lire le contenu de ses vidéos privées

Ça serait top
Merci
Noyeux Joel

**Comment by sizanic** [comment]
Salut,

L'addon "google Drive" pour kodi existe déjà et est disponible sur le dépot officiel de kodi 🤔
